.github/workflows/ci.yml

```diff
@@ -258,11 +258,22 @@ jobs:
         run: npm test &> /dev/null || exit 0
 
       - name: Accept baselines
-        run: npx hereby baseline-accept
+        run: |
+          npx hereby baseline-accept
+          git add tests/baselines/reference
 
-      - name: Check for unused baselines
+      - name: Check baselines
         run: |
-          if ! git diff --exit-code --quiet; then
-            echo "Unused baselines:"
-            git diff --exit-code --name-only
+          function print_diff() {
+            if ! git diff --staged --exit-code --quiet --diff-filter=$1; then
+              echo "$2:"
+              git diff --staged --name-only --diff-filter=$1
+            fi
+          }
+
+          if ! git diff --staged --exit-code --quiet; then
+            print_diff ACR "Missing baselines"
+            print_diff MTUXB "Modified baselines"
+            print_diff D "Unused baselines"
+            exit 1
```

Comment:
Out of curiosity, why `exit 1` here?

Reply:
If we don't do this, the step won't fail.